tests/integration/test_groupby.py

```diff
@@ -366,35 +389,38 @@ def test_groupby_split_out(c, input_table, split_out, request):
         config_options={"sql.groupby.split_out": split_out},
     )
     expected_df = (
-        user_table.groupby(by="user_id").agg({"b": "sum"}).reset_index(drop=False)
+        user_table.groupby(by="user_id")
+        .agg({"b": "sum"})
+        .reset_index(drop=False)
+        .rename(columns={"b": "S"})
+        .sort_values("user_id")
     )
-    expected_df = expected_df.rename(columns={"b": "S"})
-    expected_df = expected_df.sort_values("user_id")
-    assert df.npartitions == split_out if split_out else 1
-    dd.assert_eq(df.compute().sort_values("user_id"), expected_df, check_index=False)
+
+    assert return_df.npartitions == split_out if split_out else 1
+    assert_eq(return_df.sort_values("user_id"), expected_df, check_index=False)
 
 
+@pytest.mark.skip(reason="WIP DataFusion")
```

Comment:
Same

Reply:
The name of the PR is quite misleading. While its name is "Datafusion aggregate" in reality it is the first PR for baseline removing all Java logic and doesn't necessarily mean that all `aggregate` functionality will work along with it yet. These tests have pieces that still will not work until subsequent PRs.